docs/tutorial/part-two/index.md

```diff
@@ -298,36 +298,36 @@ export default () => (
 )
 ```
 
-> Tip: Generally, if you use a component in multiple places on a site, it should be in its own module file in the `components` directory. But, if it's used only in one file, create it inline.
+> Wskazówka: Generalnie, jeśli użyjesz komponentu w kilku różnych miejscach na stronie, powinien on zawierać się we własnym pliku module w folderze 'components'. Ale gdy jest użyty tylko w jednym pliku, napisz go "w linii".
```

Comment:
Zgadzam się z powyższym, ale dodałbym jeszcze back-ticks, do słów które nawiązują do kodu. 
```suggestion
> Wskazówka: Generalnie, jeśli użyjesz komponentu w kilku różnych miejscach na stronie, powinien on zawierać się we własnym pliku `module` w folderze `components`. Ale gdy jest użyty tylko w jednym pliku, napisz go "w linii".
```
(w oryginalnym tekście, słowo "module" nie jest zawarte w back-ticks, ale jeśli go nie tłumaczymy, to wydaje mi się że powinniśmy je dodać.)

Można też zrobić to tak (spójnie z naszym tłumaczeniem słowa [Moduł CSS](https://github.com/gatsbyjs/gatsby-pl/pull/7/files#diff-56e3fb760cff8a5eadfc58a75b142c88R140)):
```suggestion
> Wskazówka: Generalnie, jeśli zamierzasz używać komponentu w kilku różnych miejscach na stronie, powinien on zawierać się we własnym Module CSS w folderze `components`. Ale gdy jest użyty tylko w jednym pliku, napisz go "w linii".
```
Dajcie znać, co myślicie. (Osobiście preferuję moją drugą sugestię) @lekterable @matborowiak

Reply:
Podoba mi sie Twój poczatek zdania. Co robimy kiedy sugestje sa 'outdated' ???

Reply:
```suggestion
> Wskazówka: Generalnie, jeśli zamierzasz używać komponentu w kilku różnych miejscach na stronie, powinien on znajdować się we własnym Module CSS w folderze `components`. Ale jeśli jest użyty tylko w jednym pliku, umieść go bezpośrednio w kodzie.
```

hm?

Reply:
@matborowiak myślę, że najlepiej będzie jeśli zedytujesz to ręcznie i wkleisz odnośnik z linią kodu przez rozwiązaniem konwersacji

Reply:
@matborowiak albo można otworzyć nową sugestie dla ułatwienia: https://github.com/gatsbyjs/gatsby-pl/pull/7/files#r341819439